features/multimedia.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Multimedia
+  In order to mange Multimedia
+  I need abe able to create and retrieve through the API.
+  Scenario: Upload new multimedia file
```

Comment:
Test scenario with:
- without auth token
- without uploaded file
- with empty file (we should valid it?)

Reply:
done